spec/tbd_prototype_suite_spec.rb

```diff
@@ -30,6 +31,32 @@ def get_clean_env
   # number of processors to use
   nproc = [1, Parallel.processor_count - 2].max
 
+  # Fetch 'OpenStudio Results' measure (if missing).
```

Comment:
(Reiterating) NREL OpenStudio measures as required spec gems (instead of maintaining static versions within TBD). A ```bundle update``` should be sufficient to retrieve fixes by NREL, updated copyright years, etc. Needed some OSW and gemspec tweaks.